### PR TITLE
Plant CI

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

**Comment by lucafavatella** [comment]
It looks like Travis is not enabled in this repo. @elbrujohalcon could you enable please?

**Comment by elbrujohalcon** [comment]
On it!

**Comment by elbrujohalcon** [comment]
done, @lucafavatella

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [![Stories in Ready](https://badge.waffle.io/inaka/sumo_db.png?label=ready&title=Ready)](https://waffle.io/inaka/sumo_db)
 # sumo_db
 
+[![Build Status](https://travis-ci.org/inaka/sumo_db.svg?branch=master)](https://travis-ci.org/inaka/sumo_db)
```
Comment (elbrujohalcon) [inline README.md]: should be sumo_rest, not sumo_db

Comment (lucafavatella) [inline README.md]: @elbrujohalcon I believe sumo_db rather. Could you double check please?

**Comment by lucafavatella** [comment]
Closed and reopened for triggering Travis CI.

File: README.md (hunk 1 above)
Comment (elbrujohalcon) [inline README.md]: my bad entirely 😊

**Comment by elbrujohalcon** [comment]
@lucafavatella fixed… now Travis is enabled in sumo_db… as it is supposed to be

**Comment by lucafavatella** [comment]
yeaaaa Travis CI got triggered.

**Comment by lucafavatella** [comment]
... and I eye-balled the build logs of Travis are good. Ready to merge.

**Pull request merged**